2024/wg-fedid.html

```diff
@@ -150,48 +151,34 @@
     <div id="background" class="background">
       <h2>Motivation and Background</h2>
       <p>
-        Identity on the Web is critical to online interaction, with many privacy and security implications. 
-      <p>
-      <p>
-        The W3C fosters an ecosystem that addresses privacy, security, and user sovereignty.
-      </p>
+        Identity on the Web is critical to online interaction, with implications for privacy, security, and user sovereignty. 
       <p>
-        This includes developing new mechanisms that allow individuals to select identity information relevant to a given interaction, such as assertions, specific credentials, or specific attributes, supporting an open ecosystem.
       <p>
+        Safely enabling these identity-related interactions requires new mechanisms mediated by the user agent that allow individuals to select identity information relevant to a given interaction, such as assertions, credentials, or specific attributes. 
       <p>
-        These mechanisms must also be viable for issuers, identity providers, verifiers, and relying parties to exchange information securely and as privately as possible.
-      </p>
       <p>
-        The user agent is the mediator of these transactions.
```

Comment:
Did this line get deleted accidentally?

Reply:
I moved "mediated by the user agent" to a phrase in the previous paragraph.